runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -204,6 +221,30 @@ public void onGameTick(GameTick tick)
 			return;
 		}
 
+		for (DriftNet net : NETS)
+		{
+			DriftNetStatus netStatus = net.getStatus();
+			DriftNetStatus prevTickNetStatus = net.getPrevTickStatus();
```

Comment:
Why do we care about the net status of the previous tick?

Reply:
When a fish shoal is 2 tiles adjacent to a drift net and is moving to a net that is just being setup it will be denied even though the net is currently in the CATCHING status. The easiest solution seemed to be to just look to the previous tick.